scripts/install

```diff
@@ -309,7 +286,7 @@ fi
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..."
 
-yunohost service add $service_name --description "Creates a Wi-Fi access point"
+yunohost service add $service_name --description "Creates a Wi-Fi access point" --test_status "systemctl is-active hostapd" --needs_exposed_ports 547 67
```

Comment:
Uuh you sure that `--needs_exposed_ports A B` works ? Don't we need maybe something like `--needs_exposed_ports "A B"`  ? (I don't know)

Reply:
Yes ! I will do that.

Reply:
Donc non
```log
Info: [################+...] > Integrating service in YunoHost...
Warning:                             [-h] [-d DESCRIPTION] [-l LOG [LOG ...]]
Warning:                             [-t {file,systemd} [{file,systemd} ...]]
Warning:                             [--test_status TEST_STATUS]
Warning:                             [--test_conf TEST_CONF]
Warning:                             [--needs_exposed_ports PORT [PORT ...]] [-n]
Warning:                             [-s STATUS]
Warning: yunohost service add: error: argument --needs_exposed_ports: invalid int value: '547 67'
Error: Unable to install hotspot: An error occurred inside the app installation script
```